uk_bin_collection/uk_bin_collection/councils/DurhamCouncil.py

```
@@ -3,28 +3,81 @@
+    }
+    resp = requests.post(AJAX_URL, json=payload, timeout=30)
+    resp.raise_for_status()
+    result_xml = resp.json().get("result", "")
+
+    uprns = re.findall(r"<uprn[^>]*>([^<]+)</uprn>", result_xml)
+    addrs = re.findall(r"<formatted_address[^>]*>([^<]+)</formatted_address>", result_xml)
+    if not uprns:
+        raise ValueError(f"No addresses found for postcode: {postcode}")
+
+    entries = []
+    for u, a in zip(uprns, addrs):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)